_redirects

```diff
@@ -25,6 +25,8 @@ layout: null
 /docs/v21.2/migrate-from-oracle.html /docs/v21.2/migration-overview.html 302
 
 # Renamed pages
+/docs/v22.1/experimental-features.html /docs/v22.1/features-in-preview.html 301
```

Comment:
Can we also add redirects for `dev` and `stable` here?

Reply:
Done!